CHANGELOG.md

```diff
@@ -1,6 +1,34 @@
 # Change Log
 
-## [v0.6.5](https://github.com/SeldonIO/alibi/tree/v0.6.4) (2022-03-18)
+## [v0.7.0](https://github.com/SeldonIO/alibi/tree/v0.7.0) (2022-05-[TODO])
+[Full Changelog](https://github.com/SeldonIO/alibi/compare/v0.6.5...v0.7.0)
+
+This release introduces two new methods, a `GradientSimilarity` explainer and a `ProtoSelect` data summarisation algorithm.
+
+## Added
+- **New feature** `GradientSimilarity` explainer for explaining predictions of gradient-based (PyTorch and TensorFlow) models by returning the most similar training data points from the point of view of the model ([docs](TODO)).
```

Comment:
Are these still TODO's because the similarity and prototype docs have only just been merged? If its because the docs aren't in `stable` yet, maybe future convention could be to copy & paste in `latest` links and then just change `latest -> stable`?

Reply:
Yes, I wrote this when there were no links. Will also need to be `stable`.